### DISCUSSION TITLE
Importing evokes Matlab GUI and finishes fruitlessly

### DISCUSSION THREAD

**Discussion by aso-toshihiko** [ExploreASL/ExploreASL#1227] [Q&A]
Hi, 

Version 0.3.0 was working fine for me last month, but now I see this prompt pointing to strange locations of rawdata and log directories. Matlab GUI is evoked with the splashscreen and nothing else happens, just like I typed matlab and enter on CMD console.
There are no directories created under the data directories either, meaning no logs to check.

Because the only change I made was installation of Matlab2020a after the previous (successful) attempt with Matlab2022a, I changed the Windows PATH to evoke Matlab2020a or 2022a with no luck. Matlab paths are restored to the default for both versions, which is inconvenient for me!

Any idea?

![image](https://user-images.githubusercontent.com/55236088/199448473-6e51f1cc-8738-488e-a362-8255bb54310e.png)
![image](https://user-images.githubusercontent.com/55236088/199448954-0a7eabe0-ba74-4ccc-86d8-f120cb0769c9.png)

**Comment by aso-toshihiko** [comment]
Importing somehow worked by completely removing the path for Matlab2020a to force usage of 2022a. Strange. I did re-installing ExploreASL-GUI each time I made these modifications and this was the only solution.
It's Windows10 by the way. Still waiting for explanations!

**Reply by aso-toshihiko** [reply]
Let me confirm that this incorrect directory description in the prompt had nothing to do with my problem.

![image](https://user-images.githubusercontent.com/55236088/199465065-ccd0a714-72cb-4a8d-be49-d5a77f67dd12.png)

**Comment by MauricePasternak** [comment]
I will look into the behavior of having multiple MATLAB installations so see what the nature of this quirk is.

That being said, thank you for pointing out the incorrect filepaths displayed. It will be corrected in v0.4.0.
